pkg/accounting/tally/tally.go

```diff
@@ -142,9 +143,13 @@ func (t *Service) CalculateAtRestData(ctx context.Context) (latestTally time.Tim
 					bucketID := storj.JoinPaths(project, bucketName)
 
 					bucketTally := bucketTallies[bucketID]
+					projectID, err := uuid.Parse(project)
+					if err != nil {
+						return err
+					}
 					if bucketTally == nil {
 						bucketTally = &accounting.BucketTally{}
-						bucketTally.ProjectID = []byte(project)
+						bucketTally.ProjectID = projectID[:]
```

Comment:
Why is ProjectID here not `uuid`?

Reply:
I attempted to change it to uuid here, but it became kind of messy so I decided to leave it at []byte

Reply:
Add a note somewhere that this needs to be fixed.